src/server/run_sched.c

```diff
@@ -319,6 +319,15 @@ contact_sched(int cmd, char *jobid, pbs_net_t pbs_scheduler_addr, unsigned int p
 
 	net_add_close_func(sock, scheduler_close);
 
+	if (set_nodelay(sock) == -1) {
+#ifdef WIN32
+		errno = WSAGetLastError();
+#endif
+		snprintf(log_buffer, sizeof(log_buffer), "run_sched: cannot set nodelay on connection %d (errno=%d)\n", sock, errno);
```

Comment:
I don't think you need the file name in the log message here. \_\_func\_\_ will log the function name and that's what we need.